.azure-pipelines/verify.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+---
+trigger: none
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - master
+      - egeria-release-*
+
+stages:
+  - stage: Verify
+    jobs:
+      - job: VerifyBuild
+        pool:
+          vmImage: 'ubuntu-latest'
+        strategy:
+          matrix:
+            Java8:
+              JAVA_VERSION: '1.8'
+            Java11:
+              JAVA_VERSION: '1.11'
+        steps:
+          - task: Maven@3
+            inputs:
+              mavenPomFile: 'pom.xml'
+              javaHomeOption: 'JDKVersion'
+              jdkVersionOption: $(JAVA_VERSION)
+              jdkArchitectureOption: 'x64'
+              goals: 'clean deploy'
+              options: >-
+                  --batch-mode
+                  -e
+                  -DskipFVT
+                  -Dstream=$(System.PullRequest.TargetBranch)
+                  -DaltDeploymentRepository=staging::default::file:$(Agent.BuildDirectory)/local-repo
+                  -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+                  -Dmaven.source.skip=true
+          - publish: target
+            artifact: target-$(Build.BuildId)_Java-$(JAVA_VERSION)
```

Comment:
with us running java 8 and 11, does this mean they'll go to a different location in jfrog (never used) - will the maven artifacts still be named the same? For now whilst we'll build with 8 and 11, the official/final version we put on maven central etc needs to be from 8.

Reply:
Currently only the Java 8 builds are pushed to Artifactory - I've don't know yet how to manage multiple JDK version builds of projects there. These 'publish' steps will actually store them in Azure Pipelines though, the 'ArtifactoryMaven' task in the merge.yml file is what publishes them to Aritfactory.

As an example, you'd see them as target-1234_Java-1.8 under a location like: https://dev.azure.com/ODPi/Egeria/_build/results?buildId=186&view=artifacts

Reply:
ok that's fine for now. will open up new issue when we're ready to commit to 11 only